healpy/src/_sphtools.pyx

```diff
@@ -535,21 +547,30 @@ def rotate_alm(alm not None, double psi, double theta, double phi, lmax=None,
     # results.
     if len(alm) not in (1, 3):
         for a in alm:
-            rotate_alm(a, psi, theta, phi)
+            if matrix is None:
```

Comment:
I'm confused by the different ways to call rotate_alm(). Does cython allow C++-like overloading?

Reply:
yes